examples/viewer.js

```diff
@@ -36,16 +38,239 @@ let droppedFramePerArray = [];
+        },
+        offAnswerTime: {
+            name: 'sdp-exchange-master',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken to respond to an offer from the viewer with an answer',
+            color: '#FF6F00',
+        },
+        describeChannel: {
+            name: 'signaling-master-describe-channel',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken for the API call to desribeSignalingChannel on the master',
+            color: '#EF9A9A',
+        },
+        channelEndpoint: {
+            name: 'signaling-master-get-signaling-channel-endpoint',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken for the API call to getSignalingChannelEndpoint on the master',
+            color: '#EF9A9A',
+        },
+        iceServerConfig: {
+            name: 'signaling-master-get-ice-server-config',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken for the API call to getIceServerConfig on the master',
+            color: '#EF9A9A',
+        },
+        getToken: {
+            name: 'signaling-master-get-token',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken for the getToken call on the master',
+            color: '#EF9A9A',
+        },
+        createChannel: {
+            name: 'signaling-master-create-channel',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken createChannel API call on the master',
+            color: '#EF9A9A',
+        },
+        connectAsMaster: {
+            name: 'signaling-master-connect',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken for the signaling connect on the master',
+            color: '#EF9A9A',
+        },
+        iceGathering: {
+            name: 'ice-gathering-master',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken to gather all ice candidates on the master',
+            color: '#90CAF9',
+        }, 
+        peerConnection: {
+            name: 'pc-establishment-master',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken to establish the peer connection on the master',
+            color: '#2196F3',
+        },
+        dataChannel: {
+            name: 'datachannel-master',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time taken to send a message to the viewer and receive a response back',
+            color: '#4CAF50',
+        },
+        ttffAfterPc: {
+            name: 'ttff-after-pc-master',
+            startTime: '',
+            endTime: '',
+            tooltip: 'Time to first frame after the master\'s peer connection has been established',
+            color: '#2196F3',
+        }
+    }
+};
+
+let dataChannelLatencyCalcMessage = {
+    content: 'Opened data channel by viewer',
+    timestamp1: '',
```

Comment:
Lets be more descriptive in the naming here instead of timestamp1,2,3...

Reply:
Done!